ui/src/application/Applications.tsx

```diff
@@ -172,6 +187,17 @@ const Row: SFC<IRowProps> = observer(
             <TableCell>
                 <CopyableSecret value={value} style={{display: 'flex', alignItems: 'center'}} />
             </TableCell>
+            <TableCell>
```

Comment:
I prefer having the last_used column in the column before the controls (delete/edit).